src/tests/ftest/util/osa_utils.py

```diff
@@ -6,17 +6,27 @@
 """
 import ctypes
 import time
+import threading
 
 from avocado import fail_on
 from ior_test_base import IorTestBase
+from mdtest_test_base import MdtestBase
```

Comment:
(pylint-syntax-error) Cannot import 'mdtest_test_base' due to syntax error 'invalid syntax (<unknown>, line 31)'